features/base64encodedecode.yml

```diff
@@ -0,0 +1,7 @@
+name: Base64 encoding and decoding
+description: Utility functions for encoding and decoding strings to and from base 64.
+spec: https://html.spec.whatwg.org/multipage/webappapis.html#atob
+caniuse: atob-btoa
+compat_features:
+  - api.atob
+  - api.btoa
```

Comment:
For posterity: https://stackoverflow.com/questions/33854103/why-were-javascript-atob-and-btoa-named-like-that

Reply:
THANKS BRENDAN